### DISCUSSION TITLE
https://laterre.dev/posts/%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%B2%AB-%ED%8F%AC%EC%8A%A4%ED%8A%B8-%EC%9E%91%EC%84%B1/

### DISCUSSION THREAD

**Discussion by giscus** [laterre39/laterre.dev#2] [Comments]
# 블로그 첫 포스트 작성! · 𝗟𝗔𝗧𝗘𝗥𝗥𝗘 𝗕𝗟𝗢𝗚

Develop Experience @Backend Developer

https://laterre.dev/posts/%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%B2%AB-%ED%8F%AC%EC%8A%A4%ED%8A%B8-%EC%9E%91%EC%84%B1/

**Comment by laterre39** [comment]
댓글 테스트 작성

**Reply by laterre39** [reply]
대댓글 테스트

**Comment by metishonora** [comment]
안녕하세요 ㅎㅎ
우테코 디코에서 보고 왔는데 테마 이뻐요 :heart:
시간 나시면 제 [블로그](https://metishonora.github.io)도 놀러오세용 :joy:

**Reply by laterre39** [reply]
반갑습니다 동기님 블로그 방문했는데 이쁘네요 
Jekyll 기반이군요 JamStack에서 한번 고려해봤던 솔루션입니다.

근데 휴고 써보시는게 어떤가요 권장드립니다 테마 진짜 이쁜거 많습니다.

그리고 저만 링크 누르면 gitcus로 이동되는건지... 이상하네요
[블로그 링크 테스트](https://laterre.dev/posts/%EB%B8%94%EB%A1%9C%EA%B7%B8-%EC%B2%AB-%ED%8F%AC%EC%8A%A4%ED%8A%B8-%EC%9E%91%EC%84%B1/)
엥 정상작동 하는데 이상하네...

**Reply by metishonora** [reply]
제가 링크를 잘못 달았네요 https:// 를 안 붙였더니 하위 링크로 이동하네요 ㅋㅋ

**Reply by laterre39** [reply]
앗 ㅋㅋ 그렇군요 수정이 가능합니다 깃 디스커션에 오시면...

**Reply by metishonora** [reply]
아하 댓글이 discussion에 연결되어 있었군요
제 블로그는 issue에 달아놨어서, 안 그래도 issue 탭 쪽만 찾아보고 있었네요 ㅋㅋㅋㅋ 😹

**Reply by laterre39** [reply]
utteranc도 좋지만 giscus를 추천드려요 기능이 더욱 좋거든요!

**Comment by ddoddii** [comment]
안녕하세요 ! Hugo 블로그 소개글 올린 장본인..(?) 입니다 !! 
제 글 보고 이렇게 예쁘게 블로그 세팅하셨다니 도움이 되어 기쁘네요 !! 🥰 
파워 블로거까지 화이팅입니다 ~~!!

**Reply by laterre39** [reply]
글보고 휴고 선택하게 됬어요 테마들이 다른 JAMstack에 비하면 좋아요
